app/services/obituary_service.rb

```diff
@@ -21,15 +21,20 @@ def get_recent_obituaries
     JSON.parse(resp.body, symbolize_names: true)
   end
 
-  def get_advanced_obituaries(name, year)
+  def advanced_search_name_and_year(name, year)
     resp = conn.get('/advanced') do |req|
       req.params['name'] = "#{name}"
       req.params['date'] = year.to_i
     end
     JSON.parse(resp.body, symbolize_names: true)
   end
 
-
+  def advanced_search_year(year)
+    resp = conn.get('/advanced/year') do |req|
```

Comment:
Jack hopped into the Microservice to help create this end point using all the code that was already written.

Reply:
Nice teamwork!

Reply:
It might make sense to change the names of the routes in the microservice to be a little more Resful. We can talk about it later this morning.